src/main/resources/site/parts/contact/contact.xml

```diff
@@ -9,5 +9,12 @@
         <max-length>30</max-length>
       </config>
     </input>
+    <input name="contacts" type="CustomSelector">
```

Comment:
Could the contacts _mixin_ be used here ?

Reply:
I'll fix that :)